routing/route/route_test.go

```diff
@@ -188,6 +188,37 @@ func TestPayloadSize(t *testing.T) {
 			},
 			Metadata: []byte{10, 11},
 		},
+		{
```

Comment:
I think if we wanna mirror a payload-size with the "normal" blinded route, we should define the Introduction point (meaning that it should have the blinded-point in its onion) and the last hop should not have the blinded point in it, because it receives in the update_htlc record and the above hop would need a channelid too because it's not the last hop anymore?